### PR TITLE
Add snflk/bq mixed suite

### PR DESCRIPTION
**Pull request by romanbracinik**
Jira: [BIG-38](https://keboola.atlassian.net/browse/BIG-38), [BIG-39](https://keboola.atlassian.net/browse/BIG-39)
MainPr: https://github.com/keboola/connection/pull/4098

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

### CODE REVIEW

File: tests/Backend/MixedSnowflakeBigquery/SharingTest.php [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+            case 'shareBucketToProjects':
+                $targetProjectId = $this->_client2->verifyToken()['owner']['id'];
+                $this->_client->shareBucketToProjects($bucketId, [$targetProjectId], $async);
+                break;
+            case 'shareBucketToUsers':
+                $targetUser = $this->_client2->verifyToken()['admin'];
+                $this->_client->shareBucketToUsers($bucketId, [$targetUser['id']], $async);
+                break;
+            default:
+                $this->_client->$shareMethod($bucketId, $async ?? []);
```
Comment (jirkasemmler) [inline tests/Backend/MixedSnowflakeBigquery/SharingTest.php:51]: huh, to je zas chutovka. dej tam prosim malej komentar :)

File: tests/Backend/MixedSnowflakeBigquery/SharingTest.php [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+            true
+        );
+
+        /** @var BigQueryClient $wsDb */
+        $wsDb = $this->getDbConnection($workspace['connection']);
+        self::assertInstanceOf(BigQueryClient::class, $wsDb);
+
+        $wsBackend = WorkspaceBackendFactory::createWorkspaceBackend($workspace);
+
+        // create table in WS and insert some data
```
Comment (jirkasemmler) [inline tests/Backend/MixedSnowflakeBigquery/SharingTest.php:80]: tydle komentare jsou super, u tohoto testu docela nutnosti, muzes prosim stejne popsat dalsi kroky dole? Dobre se mi podle toho napisou TD testy, bo je budu muset upravit

**Review by jirkasemmler — CHANGES_REQUESTED** [review]
malickosti

File: tests/Backend/MixedSnowflakeBigquery/SharingTest.php (hunk 2 above)
Comment (romanbracinik) [reply tests/Backend/MixedSnowflakeBigquery/SharingTest.php:83]: V podstate v TD ti bude staci pridat ten dataProvider a ten switch vsekto ostatne kontrolujes v tych testoch TD ale pridal som tam komentare

**Review by jirkasemmler — APPROVED** [review]
👍

**Comment by romanbracinik** [comment]
rebase

**Pull request merged**